src/Apps/Order/Components/ExpressCheckout/ExpressCheckoutUI.tsx

```diff
@@ -352,6 +354,9 @@ export const ExpressCheckoutUI = ({ order }: ExpressCheckoutUIProps) => {
       validateAndExtractOrderResponse(
         submitOrderResult.submitOrder?.orderOrError,
       )
+
+      // Redirect to status page after successful order submission
+      router.push(`/orders/${orderData.internalID}/status`)
```

Comment:
Not a Relay expert but I think Express Checkout operates on a different order than the existing checkout flow. I wonder if the redirection here would cause some weirdness of the complex redirection logic we currently have.

Reply:
I believe this `internalId` is the artsy order Id which is what we want to use for redirect.  It is also the id we use in mutation right above. I also tested locally and that seems to work. But if you think there is a better prop to use - happy to do that. 

I guess alternatively we can just force refresh and that will do a redirect because status of the order changed but I liked the more explicit variant here.